### ISSUE TITLE
Expose CLI RPC in addition to HTTP

### ISSUE THREAD

**Issue by thenonameguy** [F1bonacc1/process-compose#15]
While transforming repositories from `docker-compose` to `process-compose` a new need came up:
Replace `docker-compose restart $SERVICE` with the `process-compose` equivalent in a bash setting.

Currently you need to  do something like:
```
curl -X 'PATCH' 'http://localhost:$PORT/process/stop/$SERVICE' 
curl -X 'PATCH' 'http://localhost:$PORT/process/start/$SERVICE' 
```
Given that process-compose is not a singleton daemon, you need to somehow differentiate between the potentially many processes running on the given host.
Given that Go already includes a HTTP client, it would make sense to provide a CLI interface that calls the exposed HTTP interfaces.

Something like:
```
-h --host String default: localhost
-p --port String required
-o --output json|text default json (no work needed translating the HTTP interface)

process-compose -p $PORT ps # /processes
process-compose -p $PORT process stop 
process-compose -p $PORT process start 
```

Additionally, a bit unrelated it would be awesome to:
- Expose an atomic/blocking restart operation
- Redirect GET / to /swagger/index.html

**Comment by F1bonacc1** [comment]
Makes sense, @thenonameguy.
I'll look into it.

**Comment by F1bonacc1** [comment]
Good Evening @thenonameguy 

Thank you for all the feedback and for the code review! I really appreciate it!
I published the v0.20.0 release. Please don't hesitate to let me know if you spot something unusual.
